modules/340-extended-monitoring/images/extended-monitoring-exporter/src/extended-monitoring.go

```diff
@@ -0,0 +1,232 @@
+package main
+
+import (
+	"context"
```

Comment:
you need to take into account the changes in the documentation, but only after testing the new exporter variant. https://github.com/Horiodino/deckhouse/tree/extended-monitoring-alerts-separately-%231311/modules/340-extended-monitoring/docs